Documentation/VSCodeConfiguration.md

```diff
@@ -4,20 +4,44 @@ Visual Studio Code does not work optimally for Serenity development, and there's
 
 The WSL Remote extension allows you to use VSCode in Windows while using the normal WSL workflow. This works surprisingly well, but for code comprehension speed you should put the Serenity directory on your WSL root partition.
 
-## Code comprehension
+- [Visual Studio Code Project Configuration](#visual-studio-code-project-configuration)
+  - [Code comprehension](#code-comprehension)
+    - [clangd](#clangd)
+    - [Microsoft C/C++ tools](#microsoft-cc-tools)
+  - [DSL syntax highlighting](#dsl-syntax-highlighting)
+  - [Formatting](#formatting)
+  - [Settings](#settings)
+  - [Customization](#customization)
+    - [Custom Tasks](#custom-tasks)
+    - [License snippet](#license-snippet)
 
-Both C++ comprehension tools listed below report fake errors.
+## Code comprehension
 
 ### clangd
 
-The official clangd extension can be used for C++ comprehension. You'll have to use the following .clangd:
+The clangd extension (`llvm-vs-code-extensions.vscode-clangd`) can be used for C++ comprehension. 
+
+You'll have to use the following `.clangd`:
 
 ```yaml
 CompileFlags:
+  Add: 
```

Comment:
I think a better recommendation would be to build the clang Toolchain with clangd enabled in Toolchain/CMake/LLVMConfig.cmake, and pass Toolchain/Local/clang/bin/clangd (or the correct path) as the clangd binary. That way if you specify Build/i686clang as build directory you don't have to manually add C++ STL paths and don't have to manually set __serenity__ because it will understand the --target directives from the compile commands JSON.

Reply:
We did strip down unnecessary paths and also added per-toolchain clangd configuration. 
It was mentioned to maybe use a script to generate this `.clang` file per toolchain.

Reply:
The recommendation you made works for developers using clang toolchain, so better declaring a fix for both toolchain users.